### PR TITLE
VZ-1741.  Adopt new profiles implementation in script installer

### PR DESCRIPTION
**Pull request by michael-cico**
Update installer and values files to use new settings for dev/prod profiles.

Depends on
https://github.com/verrazzano/verrazzano-operator/pull/205
https://github.com/verrazzano/verrazzano-monitoring-operator/pull/49

### CODE REVIEW

File: install/chart/values.yaml [hunk 1]
```diff
@@ -31,15 +32,20 @@ verrazzanoOperator:
   esMasterNodeRequestMemory: 1.4Gi
   esIngestNodeRequestMemory: 2.5Gi
   esDataNodeRequestMemory: 4.8Gi
+  esMasterNodeReplicas: 3
```
Comment (gilbode) [inline install/chart/values.yaml:35]: I'm not sure I like the structure of this - just a big flat list of properties under Verrazzano operator.  Seems you are trying to configure 2 different things here - whether and how we create 1 or more Elasticsearch environments for the user - and whether or how we connect the application to it.  I'd like to revisit this before we make this end user facing.

**Review by agv426 — APPROVED** [review]
Looks good to me, I'm assuming there will be more evolving here around Mike Gilbode's comment (ie: structure may change before it is public)

File: install/chart/values.yaml (hunk 1 above)
Comment (michael-cico) [reply install/chart/values.yaml:35]: OK...the other question is how much the values approach will survive the shift to the new config and the in-cluster installer, and what will be exposed.  When I talked with @desagar and @pfmackin on Tuesday, it sounds like these values files will not be exposed.

That being said, I can take another crack at making these more hierarchical.  Did you want things grouped by say, ES vs. Prometheus vs. Kibana, for example? That's kind of how the VMI is organized also.

I think at some point we're either going to end up exposing the VMI through a binding, which might line up with that thinking, or exposing something like the CRDs for prometheus/ES operators if we end up switching to those and retiring the VMO, etc.

Comment (michael-cico) [reply install/chart/values.yaml]: Updated per our meeting, and tested locally.

**Review by desagar — APPROVED** [review]
lgtm

**Pull request merged**